API/Entities/Operation.cs

```diff
@@ -1,21 +1,30 @@
 using System;
-
+using System.ComponentModel.DataAnnotations;
 namespace API.Entities
 {
     public class Operation
     {
-        public double Amount { get; set; }
+        public int Id { get; set; }
 
-        public DateTime Date { get; set; } = DateTime.Now;
+        [Required]
+        [MaxLength(100)]
+        public string Description { get; set; }
+        
+        public decimal Amount { get; set; }
 
-        public int Id { get; set; }
+        [Required]
```

Comment:
Sprwadz czy jak zmienisz i tu wywalisz required iz robisz nowa migracje to czy cos sie zmieni. Jak nie to dla DateTime jest tak samo jak dla intow - domyslnie sa wymagane bo jako typ nie przyjmuja nulla (dateTime to struct jest a nie klasa)